addon/-private/system/store.js

```diff
@@ -2432,6 +2432,16 @@ function _commit(adapter, store, operation, snapshot) {
       if (adapterPayload) {
         payload = normalizeResponseHelper(serializer, store, typeClass, adapterPayload, snapshot.id, operation);
         if (payload.included) {
+          // first find possibly uncommitted records from this inclusion
+          var types = payload.included.map(function (record) { return record.type; });
+          types.forEach(function (type) {
+            var existingRecords = store.peekAll(type);
+            var uncommitted = existingRecords.filter(function (record) {
+              return record.get('isNew') && !record.get('isSaving');
```

Comment:
Looks like this will unload all record of a type the `isNew` state even if they were not the records sent in the original request?